src/artikler/sok-sosialhjelp/SokSosialhjelpBokmal.tsx

```diff
@@ -166,7 +166,7 @@ const SokSosialhjelpBokmal: React.FC = () => {
                 </Avsnitt>
             </IkkeSokDigitaltPanel>
 
-            <h3>Kom i gang med digital søknad</h3>
+            <Undertittel>Kom i gang med digital søknad</Undertittel>
```

Comment:
Skal ikke denne ha en tag, som de andre? 
```tag="h3"```

Reply:
Denne skal være h2, som er default for Undertittel. :)

Reply:
Aha, default ja. 
Så ikke det her : D
https://design.nav.no/components/typografi/technical